__tests__/app.test.js

```diff
@@ -222,7 +222,60 @@ describe("POST /api/articles/:article_id/comments", () => {
       .send(newComment)
       .expect(422)
       .then(({ body }) => {
-        expect(body.msg).toEqual("Unprocessable entity");
+        expect(body.msg).toBe("Unprocessable entity");
       });
   })
 });
+
+describe('PATCH /api/articles/:article_id', () => {
+  it('200 - responds with an object containing the updated article, with an increase in votes', () => {
+    const newArticle = {
+      inc_votes: 10
+    }
+    return request(app)
+      .patch('/api/articles/1')
+      .send(newArticle)
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body
+        expect(article.votes).toBe(110)
+      })
+  })
+  it('200 - responds with an object containing the updated article, with a decrease in votes', () => {
+    const newArticle = {
+      inc_votes: -90
+    }
+    return request(app)
+      .patch('/api/articles/1')
+      .send(newArticle)
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body
+        expect(article.votes).toBe(10)
+      })
+  })
+  it('400 - responds with an error msg for an invalid article_id type', () => {
+    const newArticle = {
+      inc_votes: 10
+    }
+    return request(app)
+      .patch('/api/articles/not-a-number')
+      .send(newArticle)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Invalid data type')
+      })
+  })
+  it('404 - responds with an error msg for an article_id that does not exist', () => {
+    const newArticle = {
+      inc_votes: 10
+    }
+    return request(app)
+    .patch('/api/articles/9999')
+    .send(newArticle)
+    .expect(404)
+    .then(({ body }) => {
+      expect(body.msg).toBe('Not found')
+    })
+  })
+})
```

Comment:
Good tests. Still two sad paths to add -
newArticle = {}. - 400  missing require key
newArticle = {inc_votes: "banana"} - 400 wrong datatype